src/components/constants/Pokemon.ts

```diff
@@ -231,12 +231,17 @@ export interface IPokemonEvolutionByStats {
     /**
      * The stat that should be larger to achieve target evolution.
      */
-    greaterStat?: string;
+    greaterStat: string;
 
     /**
      * The stat that should be smaller to achieve target evolution.
      */
-    lesserStat?: string;
+    lesserStat: string;
+    
+    /**
+     * A flag to be set if the two stats should actually be equal.
```

Comment:
"should actually be" feels off (it's a completely valid situation)... how about "may be"?